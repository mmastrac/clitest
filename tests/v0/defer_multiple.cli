#!/usr/bin/env clitest --v0

defer {
    $ echo 1a
    ! 1a
    $ echo 1b
    ! 1b
}

defer {
    $ echo 2a
    ! 2a
    $ echo 2b
    ! 2b
}
