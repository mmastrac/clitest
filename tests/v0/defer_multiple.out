Running <root>/defer_multiple.cli ...

(deferred) echo 2a
---
2a
---
✅ OK

(deferred) echo 2b
---
2b
---
✅ OK

(deferred) echo 1a
---
1a
---
✅ OK

(deferred) echo 1b
---
1b
---
✅ OK

<root>/defer_multiple.cli PASSED
